core/i18n/src/ITwinLocalization.ts

```diff
@@ -132,7 +144,7 @@ export class ITwinLocalization implements Localization {
    * @throws Error if no keys resolve to a string.
    * @internal
    */
-  public getLocalizedStringWithNamespace(namespace: string, key: string | string[], options?: TOptionsBase): string {
+  public getLocalizedStringWithNamespace(namespace: string, key: string | string[], options?: TranslationOptions): string {
```

Comment:
just mentioning, but in the getLocalizedString you added some additional handling, depending on the properties of options

Reply:
same thing in getEnglishString

Reply:
`getLocalizedStringWithNamespace()` calls `getLocalizedString()`, so the additional handling is covered. `getEnglishString()` does not... so I'll add it in there.

Reply:
^ That change is implemented now.